synapse/lib/view.py

```diff
@@ -1098,7 +1098,7 @@ async def setViewInfo(self, name, valu):
         '''
         Set a mutable view property.
         '''
-        if name not in ('name', 'desc', 'parent', 'nomerge', 'quorum'):
+        if name not in ('name', 'desc', 'parent', 'nomerge', 'protected', 'quorum'):
```

Comment:
We should be safe to remove the `nomerge` value here in 3.x.x

Reply:
Agree. I'll add a comment.